src/main.ts

```diff
@@ -43,6 +48,10 @@ export default function createAndMount(
 	const store = initStore( { ...services, languageCodesProvider } );
 	app.use( store );
 
+	if ( config.params !== undefined ) {
+		store.dispatch( INIT_FROM_PARAMS, config.params ); // without await
```

Comment:
I wonder if we want to name it differently in this app, as "params" is a very generic term. Maybe `INIT_FROM_URL` would be more to the point of what is going on. Or maybe `INIT_WITH_DATA` or just `INIT` could also be plausible, as the app doesn't really know/care where that data is coming from?

Though I'm not sure about this yet, just a thought

Reply:
I like “params” better than “data” because at least “params” is often used as the term for the URL params specifically… what if we rearrange the words a bit? We could use “init params” is a noun phrase (like in the `InitParams` interface already), and change the action name from “init from (generic) params” to something like “update from init params” or “handle init params”.

Reply:
Did that now.

Reply:
Yes, I like that more, thank you!